apps/site/next-data/downloadSnippets.ts

```diff
@@ -8,11 +8,13 @@ import {
 import { availableLocaleCodes } from '@/next.locales.mjs';
```

Comment:
Any reason why changing these files? This doesn't seem to be related? Are you just unhappy with the style?

Reply:
Could you maybe revert these changes as these are unrelated?

Reply:
I can, but this isn't just about style; using an async function is faster and cleaner than making a promise and returning it.